test/examples/runtests.jl

```diff
@@ -89,6 +90,44 @@ if "rings" in cases
     )
 end
 
+reltol = 1.0e-2
+
+if "transmon" in cases
+    @info "Testing single transmon..."
+    @time testcase(
+        "transmon",
+        "transmon.json",
+        "";
+        palace=palace,
+        np=numprocs,
+        rtol=reltol,
+        atol=abstol,
+        excluded_columns=[
+            "Maximum",
+            "Minimum",
+            "Mean",
+            "Error (Bkwd.)",
+            "Error (Abs.)",
+            "Re{V[1]} (V)",
+            "Im{V[1]} (V)",
+            "Re{V[2]} (V)",
+            "Im{V[2]} (V)",
+            "Re{V[3]} (V)",
+            "Im{V[3]} (V)",
+            "Re{I[1]} (A)",
+            "Im{I[1]} (A)",
+            "Re{I[2]} (A)",
+            "Im{I[2]} (A)",
+            "Re{I[3]} (A)",
+            "Im{I[3]} (A)"
```

Comment:
I following the `cpw_lumped_eigen` example below. Some of these were prone to failing